### ISSUE TITLE
Cannot Upload Subscribers File

### ISSUE THREAD

**Issue by JimDalton** [TransNexus/NexOSS#299]
#### Page Path
/ui/Accounts_Subscribers

#### Description of Issue
When I try to upload the attached Subscribers file, the response is Internal Server Error.
[PressOne_calling_line_list_No_Duplicates.txt](https://github.com/TransNexus/NexOSS/files/609978/PressOne_calling_line_list_No_Duplicates.txt)

**Comment by JimDalton** [comment]
Screenshot of log messages.
![image](https://cloud.githubusercontent.com/assets/18315382/20673074/a175ec6c-b551-11e6-9886-0be5f104fa97.png)

**Comment by DmitryIsakbayev** [comment]
@JimDalton, the file is rejected because some lines are missing values for the last column -  UserId.
I will fix the data validation logic.

**Comment by DmitryIsakbayev** [comment]
@fenichelar, the error message that the server generates should be more descriptive, but it sending "4" in the "X-Error-Description" header.  Here is the HTTP response.

HTTP/1.1 580 4^M
Cache-Control: no-store, no-cache, must-revalidate, max-age=0, post-check=0, pre-check=0^M
**X-Error-Description: 4**^M
Content-Type: text/html;charset=ISO-8859-1^M
Content-Language: en-US^M
Date: Mon, 28 Nov 2016 19:13:54 GMT^M
Server: Apache-Coyote/1.1^M
Connection: close^M

**Comment by DmitryIsakbayev** [comment]
Please test after the lab is upgraded to 8.18.2

**Comment by JimDalton** [comment]
This problem still exists on NexOSS V8.18.4

**Comment by JimDalton** [comment]
@DmitryIsakbayev   
[PressOne_calling_line_list_No_Duplicates, no`.txt](https://github.com/TransNexus/NexOSS/files/622877/PressOne_calling_line_list_No_Duplicates.no.txt)
I found, and removed, the ` character.  However, I can still not upload the attached file.  Is there another character that is causing a problem?

**Comment by DmitryIsakbayev** [comment]
@JimDalton, the link to file in your previous post does not work.  Please reattach the file or email it to me.

**Comment by JimDalton** [comment]
Fixed.  They problem I kept reporting was caused by MS Excel which does not identify duplicates when a value includes a trailing white space.  The lines below, identified by the NexOSS error message, reveal the second CLID has a trailing space.  Closing this issue to open a new requirement to ignore duplicates.

Pressone	Pressone	COMMX	+12122442727	AmericanCopyDemo
sp_commx	CommX-Tampa	AMERCOPY151109	+12122442727    	AMERCOPY2745
